arcgis-ios-sdk-samples/Utility network/Create load report/CreateLoadReport.storyboard

```diff
@@ -0,0 +1,103 @@
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="17703"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Create Load Report View Controller-->
+        <scene sceneID="bKB-DV-zIK">
+            <objects>
+                <viewController id="SSS-gO-RzR" customClass="CreateLoadReportViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="0T0-G9-aKP">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="0ks-Dm-NAf">
+                                <rect key="frame" x="0.0" y="813" width="414" height="49"/>
+                                <items>
+                                    <barButtonItem title="Reset" id="bhu-cS-Gsm">
+                                        <connections>
+                                            <action selector="resetBarButtonItemTapped:" destination="SSS-gO-RzR" id="Hyj-tV-4Ey"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="bnh-px-UF6"/>
+                                    <barButtonItem systemItem="add" id="bM0-pR-Lrw">
+                                        <connections>
+                                            <action selector="addBarButtonItemTapped:" destination="SSS-gO-RzR" id="YML-fw-rnH"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="xUT-hy-Cda"/>
+                                    <barButtonItem title="Run" id="5Du-wq-bZz">
+                                        <connections>
+                                            <action selector="runBarButtonItemTapped:" destination="SSS-gO-RzR" id="2kg-VK-tP2"/>
+                                        </connections>
+                                    </barButtonItem>
```

Comment:
Should all the buttons be disabled by default and then enabled as needed?

Reply:
Good point. I didn't change the states of buttons in the first place, because I thought it would introduce ~20 lines of additional code that has nothing to do with the APIs themselves as the sample is already pretty long. 

Since you mentioned it, I'll disable and enable the buttons in the code.